terraform/validator-sets/dev/seed_peers.config.toml

```diff
@@ -1,2 +1,2 @@
 [seed_peers]
-8deeeaed65f0cd7484a9e4e5ac51fbac548f2f71299a05e000156031ca78fb9f = ["/ip4/SEED_IP/tcp/6180"]
+ab0d6a54ce9d7fc79c061f95883a308f9bdfc987262b6a34a360fdd788fcd9cd = ["/ip6/::1/tcp/60420"]
```

Comment:
Can you loop with @fbandersnatch to make sure if we do want to checkin this change?

Reply:
Yeah I'm not really sure. @fbandersnatch?

Reply:
This shouldn't change. I think that we exclude it from comparison/remove it from the diff when running the circle script.